src/NodaTime.Test/LocalDateTest.cs

```diff
@@ -71,5 +71,22 @@ public void MinIsoValue()
             Assert.AreEqual(CalendarSystem.Iso, value.Calendar);
             Assert.Throws<OverflowException>(() => value.PlusDays(-1));
         }
+
+        [Test]
+        public void Deconstruction()
+        {
+            var value = new LocalDate(2017, 11, 7);
+            var expectedYear = 2017;
+            var expectedMonth = 11;
+            var expectedDay = 7;
+            value.Deconstruct(out int actualYear, out int actualMonth, out int actualDay);
+
+            Assert.Multiple(() =>
+            {
+                Assert.AreEqual(expectedYear, actualYear, "Year comparison failed.");
```

Comment:
No need for the messages here - if there are failures, we can tell what's failed by the stack trace, or probably just by the numbers.

Reply:
Got it.